src/rules/at-rule-blockless-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,27 @@
+import {
+  ruleTester,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule("always", tr => {
+  tr.ok("@import url(\"x.css\");\n")
+  tr.ok("@charset \"UTF-8\";\n")
+  tr.ok("@namespace url(XML-namespace-URL);\n")
+  tr.ok("@import url(\"x.css\"); /* comment */\n")
+  tr.ok("@import url(\"x.css\");\r\n", "windows")
+  tr.ok("@import url(\"x.css\"); /* comment */\r\n", "windows")
+  tr.ok("@import 'x.css';", "single blockless rule")
+
+  tr.notOk("@import url(\"x.css\"); @charset \"UTF-8\";", {
+    message: messages.expectedAfter(),
+    line: 1,
+    column: 22,
+  })
+  tr.notOk("@charset \"UTF-8\"; a {};", {
```

Comment:
Should tabs be allowed here? If yes, then the [tests for declaration-block-semicolon-newline-after](https://github.com/stylelint/stylelint/blob/master/src/rules/declaration-block-semicolon-newline-after/__tests__/index.js#L69) should be updated too.